packages/shared/src/components/feeds/FeedNav.tsx

```diff
@@ -108,7 +116,12 @@ function FeedNav(): ReactElement {
               onChange={(_, index) => setSelectedAlgo(index)}
               drawerProps={{ displayCloseButton: true }}
             />
-          </div>
+          </StickyNavIconWrapper>
+        )}
+        {!isMobile && !isLaptop && (
+          <StickyNavIconWrapper className="translate-x-[calc(100vw-180%)]">
+            <NotificationsBell compact />
+          </StickyNavIconWrapper>
```

Comment:
This is the only icon we need to add?
Just wondering as there are quite a few additional items right

Reply:
Though I can remove the conditional render via js and will hide in laptop and mobile via css